### ISSUE TITLE
SAVE THE FORESTS: Comment schedule and recommend for plugins writers to avoid daily builds

### ISSUE THREAD

**Issue by nugged** [bywatersolutions/dev-koha-plugin-kitchen-sink#23]
https://github.com/bywatersolutions/dev-koha-plugin-kitchen-sink/blob/108b43e50f0a0938d6def235ab9f81b0f2a87d98/.github/workflows/main.yml#L9

Why scheduled builds are not needed most of the time:
99% of plugins do not need to be built daily, so this is very unethical burning of coil for all repos and forks, daily, 5 minutes of CPU time, and usually without reason.

... please, commend this and hint all plugin writers to use scheduled build only when there is a reason for it. Moreover: scheduled build even if enabled for a reason, might also happen on forked experiments, which even more not needed.

**Comment by tomascohen** [comment]
I agree here. I think this was for keeping the project activity. But we could just run it monthly?
